scenarios/contact-menu.json

```diff
@@ -68,6 +68,50 @@
       "url": "{url}/civicrm/group?reset=1",
       "onReadyScript": "contact/manage-groups.js"
     },
+    {
+      "label": "Manage Groups - Edit Group settings modal",
+      "url": "{url}/civicrm/group?reset=1",
+      "selectors": [".ui-dialog"],
+      "onReadyScript": "contact/manage-groups-settings-modal.js"
+    },
+    {
+      "label": "Manage Groups - Add Contacts screen - results",
+      "url": "{url}/civicrm/group?reset=1",
+      "onReadyScript": "contact/manage-groups-edit-group-add-contacts.js"
+    },
+    {
+      "label": "Manage Groups - Add Contacts to the group form - search form - results",
+      "url": "{url}/civicrm/group?reset=1",
+      "onReadyScript": "contact/manage-groups-edit-group-add-contacts-to-group-search-form.js"
```

Comment:
this filename format really seem overkill and results in unnecessarily long names. This could be simply be called `add-contacts-to-group-search-form.js`

Reply:
Cool!